app/src/main/java/project/pb/fragments/FAQFragment.java

```diff
@@ -34,69 +37,39 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
     }
 
 
-    public void fillData(){
-        Questions = new ArrayList<>();
+    public void fillData() {
         answers = new HashMap<>();
 
-        Questions.add("Can i park outside the school?");
-        Questions.add("Where is the school located?");
-        Questions.add("Is the food available at the Open day?");
-        Questions.add("What time does the Open day usually end?");
-        Questions.add("Can i park outside the school?");
-        Questions.add("Where is the school located?");
-        Questions.add("Is the food available at the Open day?");
-        Questions.add("What time does the Open day usually end?");
-        Questions.add("Can i park outside the school?");
-        Questions.add("Where is the school located?");
-        Questions.add("Is the food available at the Open day?");
-        Questions.add("What time does the Open day usually end?");
+        questions = Arrays.asList(
```

Comment:
```suggestion
        questions = Arrays.asList(
```
Im going to have a look into the .asList function, seems to prevent writing a lot of code

Reply:
Adjusted it according to your feedback